src/main/scala/ai/privado/exporter/DataflowExporter.scala

```diff
@@ -104,19 +99,23 @@ class DataflowExporter(cpg: Cpg, dataflowsMap: Map[String, Path], taggerCache: T
     }
 
     // sinkMap will have (sinkId -> List[String]() where value are all the paths/grouping-of-path which belong to the sinkId
-    val sinkMap = mutable.HashMap[String, ListBuffer[String]]()
+    val sinkMap     = mutable.HashMap[String, ListBuffer[String]]()
+    val sinkApiUrls = mutable.HashMap[String, mutable.Set[String]]()
     sourceModelList.foreach(sourceModel => {
       var sinkId = sourceModel.sinkId
       val sinkAPITag = dataflowsMap(sourceModel.pathId).elements.last.tag
         .filter(node => node.name.equals(Constants.apiUrl + sourceModel.sinkId))
-      if (sinkAPITag.nonEmpty) {
-        sinkId += "#_#" + sinkAPITag.value.l.mkString(",")
-      }
+      if (!sinkApiUrls.contains(sinkId))
+        sinkApiUrls(sinkId) = mutable.Set()
+      for (url <- sinkAPITag)
+        sinkApiUrls(sinkId).add(url.value)
```

Comment:
There should be `addAll` kind of function, use that

Reply:
Addressed here: https://github.com/Privado-Inc/privado-core/commit/4110826932d0e7852b80590ee5c5b98e91c22fdb